coopr-server/src/main/java/co/cask/coopr/upgrade/UpgradeTo0_9_9.java

```diff
@@ -0,0 +1,233 @@
+              JsonObject jsonObject = GSON.fromJson(reader, JsonObject.class);
+              String taskName = jsonObject.get("taskName").getAsString();
+              ProvisionerAction type = ProvisionerAction.valueOf(taskName.toUpperCase());
+              updateStatement.setString(1, type.name());
+
+              Cluster cluster = clustersById.get(clusterId);
+              if (cluster != null) {
+                updateStatement.setString(2, cluster.getClusterTemplate().getName()); // 2: cluster_template_name
+                updateStatement.setString(3, cluster.getAccount().getUserId()); // 3: user_id
+                updateStatement.setString(4, cluster.getAccount().getTenantId()); // 4: tenant_id
+              } else {
+                updateStatement.setString(2, null); // 2: cluster_template_name
+                updateStatement.setString(3, null); // 3: user_id
+                updateStatement.setString(4, null); // 4: tenant_id
+              }
+
+              // 5: task_num, 6: job_num, 7: cluster_id
+              updateStatement.setLong(5, taskNum);
+              updateStatement.setLong(6, jobNum);
+              updateStatement.setLong(7, clusterId);
+
+              updateStatement.addBatch();
+              indexWithinCurrentBatch++;
+
+              if (indexWithinCurrentBatch > batchSize) {
+                updateStatement.executeBatch();
+                updatesSent += (indexWithinCurrentBatch - 1);
+                LOG.info("Successfully sent batch of {} updates", indexWithinCurrentBatch - 1);
+                indexWithinCurrentBatch = 0;
+              }
+            }
+            if (indexWithinCurrentBatch != 0) {
+              updateStatement.executeBatch();
+              updatesSent += (indexWithinCurrentBatch - 1);
+              LOG.info("Successfully sent final batch of {} updates, for a total {} updates",
+                       indexWithinCurrentBatch - 1, updatesSent);
+            }
+          } finally {
+            updateStatement.close();
+          }
+        } finally {
+          rs.close();
+        }
+      } finally {
+        statement.close();
+      }
+    } finally {
+      conn.close();
+    }
+  }
+
+  public static void main(String[] args) throws ClassNotFoundException {
+    Options options = getOptions();
+    CommandLineParser parser = new BasicParser();
+    try {
+      CommandLine command = parser.parse(options, args);
+      String jdbcConnectionString = command.getOptionValue("c");
+      String jdbcDriver = command.getOptionValue("d", Driver.class.getName());
+      String dbMaxActiveConnections = command.getOptionValue("n", "100");
+      String dbUser = command.getOptionValue("u", "");
+      String dbPass = command.getOptionValue("p", "");
+      long batchSize = Long.parseLong(command.getOptionValue("b", "25000"));
+
+      final Configuration configuration = Configuration.create();
+      configuration.set(Constants.JDBC_CONNECTION_STRING, jdbcConnectionString);
```

Comment:
i think its fine to force people to use the same conf as the server instead of taking command line args. That way its guaranteed to connect to the correct db.